docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+---
+title: .NET Observability with OpenTelemetry
+description: An introduction to observing .NET apps with OpenTelemetry
+ms.date: 6/14/2023
+ms.topic: conceptual
+---
+
+# .NET observability with OpenTelemetry
+
+When you run a distributed application, you want to know how well the app is performing and to detect potential problems before they become larger. One way that you can do this is by monitoring specific instrumentation emitted by the application and looking for changes in metrics and logs.
```

Comment:
"monitoring ... instrumentation emitted ..." I don't know if these terms are well established in .NET or if there is a consistent pattern, IMHO these terms are confusing and could be misleading, might be helpful to check with @noahfalk and @tarekgh who have more context/history.

My understanding:
* "monitoring" applies to "distributed application"
* "instrumentation" cannot be "emitted", "instrumentation" is the artifact that hooks the telemetry signal/events, and an implementation (e.g. SDK) is configured to subscribe to the instrumentations and "emit" the data via some exporters.